lib/Minilla/Project.pm

```diff
@@ -122,6 +122,23 @@ sub version {
     return $version;
 }
 
+sub static_install {
+    my $self = shift;
+    my $v = exists $self->config->{static_install} ? $self->config->{static_install} : 'auto';
+    return 0+$v if $v =~ /^\d+$/;
+    errorf "Found unsupported value '%s' for static_install in minil.toml", $v if $v ne 'auto';
+
+    return 0 if $self->build_class ne 'Module::Build';
```

Comment:
Shouldn't this be inverted? (allow static install only if installer is not Module::Build)

Reply:
Oh this is checking if build_class is set to a custom class, not if the installer is Module::Build. All good!